pwnlib/context/__init__.py

```diff
@@ -1232,13 +1233,22 @@ def cache_dir(self):
             >>> cache_dir == context.cache_dir
             True
         """
+        try:
+            return self._cache_dir
+        except AttributeError:
+            pass
+
         xdg_cache_home = os.environ.get('XDG_CACHE_HOME') or \
                          os.path.join(os.path.expanduser('~'), '.cache')
 
-        if not os.access(xdg_cache_home, os.W_OK):
-            return None
-
-        cache = os.path.join(xdg_cache_home, '.pwntools-cache-%d.%d' % sys.version_info[:2])
+        if not os.access(xdg_cache_home, os.W_OK) or True:
```

Comment:
The `or True` needs to be removed

Reply:
lol, my secret testing strategy has leaked

Reply:
Gone